docs/source/diagnostics.rst

```diff
@@ -9,14 +9,11 @@ Pints provides a number of functions to diagnose MCMC progress and convergence.
 Overview:
 
 - :func:`rhat`
-- :func:`rhat_all_params`
 - :func:`effective_sample_size`
```

Comment:
Sorry, realise they weren't before, but can we make these alphabetical?

Reply:
Done